ui/components/component-library/button-primary/button-primary.stories.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { ALIGN_ITEMS, DISPLAY } from '../../../helpers/constants/design-system';
+import Box from '../../ui/box/box';
+import { ICON_NAMES } from '../icon';
+import { ButtonPrimary } from './button-primary';
+import { BUTTON_PRIMARY_SIZES } from './button-primary.constants';
+import README from './README.mdx';
+
+const marginSizeControlOptions = [
+  undefined,
+  0,
+  1,
+  2,
+  3,
+  4,
+  5,
+  6,
+  7,
+  8,
+  9,
+  10,
+  11,
+  12,
+  'auto',
+];
+
+export default {
+  title: 'Components/ComponentLibrary/ButtonPrimary',
+  id: __filename,
+  component: ButtonPrimary,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    as: {
+      control: 'select',
+      options: ['button', 'a'],
+      table: { category: 'button base props' },
+    },
+    block: {
+      control: 'boolean',
+      table: { category: 'button base props' },
+    },
+    children: {
+      control: 'text',
+    },
+    className: {
+      control: 'text',
+    },
+    danger: {
+      control: 'boolean',
+    },
+    disabled: {
+      control: 'boolean',
+      table: { category: 'button base props' },
+    },
+    icon: {
+      control: 'select',
+      options: Object.values(ICON_NAMES),
+      table: { category: 'button base props' },
+    },
+    iconPositionRight: {
+      control: 'boolean',
+      table: { category: 'button base props' },
+    },
+    iconProps: {
+      control: 'object',
```

Comment:
receiving this error in the console 

<img width="912" alt="Screenshot 2022-10-05 at 5 43 22 PM" src="https://user-images.githubusercontent.com/20778143/194159711-f18b2f7a-618d-4259-ae53-9903f34735c5.png">